docs/_documentation/using_repartitioning.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Repartitioning
+position: 12
+---
+
+## Repartitioning
+
+Throughput of event type is defined by default statistic, which basically sets number of partitions for the event type (although it does not represent it clearly). Number of partitions is a scaling unit for Nakadi publishing and consumption. In order to change number of partitions one have to repartition event type by changing read_parallelism and write_parallelism.
+
```

Comment:
Don't you think it makes sense to add that we take the maximum of read or write parallelism and to actually change they need to make sure the maximum value has to be changed?

Reply:
9a81a184